### PR TITLE
Portfolio-Website Project

### PR DESCRIPTION
**Pull request by Vishal30M**
I have finished this project as required. I have applied the features as follows:

Follow GitHub workflow.
Added works section in index.html and styled it according to the Figma template.
Added the "About myself" section placed after the works section in the index.html and styled it according to the Figma template.
Checked and set up linter errors for Html &CSS.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -174,3 +174,525 @@ header nav div .menu-bar {
...
+  flex-direction: row;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  margin-right: 8px;
+}
+
```
Comment (Owusu-Desmond) [inline style.css]: - [ ] Why are you repeating many CSS properties here for elements of the same kind? So why don't you give those element one class and use it to style them one which is acceptable as best practices but not what you have done here? Please remember the DRY(Dont respect yourself) principle. :smile:

File: index.html [hunk 2]
```diff
@@ -1,43 +1,239 @@
...
+              <li class="li11">
+                <article class="text1">html</article>
+              </li>
+              <li class="li12">
+                <article class="text2">css</article>
+              </li>
+              <li class="li13">
+                <article class="text3">javascript
+              </li>
+            </ul>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure that the design of this section matches the Figma design

### Your Design
Screenshot from 2023-01-14 00-21-32
### Figma Design
Screenshot from 2023-01-14 00-26-26

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Vishal30M , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Vishal30M** [comment]
I have made all the required changes, Please review it and let me if I need any changes more or shall I move forward.

File: index.html [hunk 3]
```diff
@@ -1,43 +1,239 @@
...
+              </li>
+              <li class="li11">
+                <article class="text2">css</article>
+              </li>
+              <li class="li11">
+                <article class="text3">javascript
+              </li>
+            </ul>
+            <div class="action">
+              <button class="btn">See Project</button>
```
Comment (Shedrack-Sunday) [inline index.html:79]: - [ ] Just as requested by the previous reviewer, kindly adjust this section of your app. you can wrap them in a `div` element, then use flexbox to render them while giving them a uniform styling. 
# Figma:
image

# Your design :
image

File: style.css [hunk 4]
```diff
@@ -174,3 +174,501 @@ header nav div .menu-bar {
...
+  display: flex;
+  flex-direction: row;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  margin-right: 8px;
+}
```
Comment (Shedrack-Sunday) [inline style.css:369]: - Just as the previous reviewer pointed out, this set of styling has been repeated a couple of times in your code. Always reference the [HTML/CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) best practices module, and keep that in mind when writing your code. please make the required change and see the beauty of `DRY` principle
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Vishal30M ,


CHANGES REQUIRED ♻️
Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



Highlight
✅All linters are checked.
✅ Proper setup
✅ Descriptive commit messages


Required Changes ♻️

- The two changes requested by the previous reviewer in comparison with the figma design section, and suggestions on following the DRY principle have been highlighted. Kindly check them in the comments, and so make the required changes. 👏 

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by nicupop729 — COMMENTED** [review]
Hello @Vishal30M 👋,

_This is an additional review as a result of the second opinion request._

Congrats on the good work done in the first week at Microverse! :fireworks: Kodos for that :clap:

In my opinion, the technologies are displayed well on the UI.
image

Regarding the DRY code, I trust that you will learn during this Module how to build a more clean and dry code.

**In my opinion, your project is ready to be approved, in accordance with what was asked from you by the reviewers.**

My optional suggestion is regarding the responsiveness of your app. At the moment, your app has a fixed design. Here at Microverse, we put the accent on the modern way of building the app, with the use of Flexbox and Grid.

The app should take the whole width of the screen from 375px to at least 750px. At this moment, this is not the case.
Kindly, take a close look into it and for the next milestone please try to have a responsive design.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

File: style.css (hunk 1 above)
Comment (bobb-Rob) [reply style.css]: It's actually _**Don't repeat yourself**_. I believe it was a typo.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Vishal30M , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**